Class 2 Exercises and Homework.js

```diff
@@ -4,41 +4,76 @@
 // - lastName
 // - 'favorite food'
 // - bestFriend (object with the same 3 properties as above)
-
+// Myself object
+const koteshObject = {
+  firstName: 'Kotesh',
+  lastName: 'Boda',
+  'favorite food': 'Mediterranean',
+  
+  bestFriend: {
+    firstName: 'Brad',
+    lastName: 'Pitt',
+    'favorite food': 'Mexican',  
+  },
+};
 
 // 2. console.log best friend's firstName and your favorite food
-
+console.log(`${koteshObject.firstName} best friend's first name is: `, koteshObject.bestFriend.firstName);
+console.log(`${koteshObject.firstName} favorite food is: `, koteshObject['favorite food']);
 
 // 3. Create an array to represent this tic-tac-toe board
 // -O-
 // -XO
 // X-X
-
+const ticTacToeArray = [['-', 'O', '-'], ['-', 'X', 'O'], ['X', '-', 'X']];
 
 // 4. After the array is created, 'O' claims the top right square.
 // Update that value.
-
+ticTacToeArray[0][1] = '-';
```

Comment:
Not sure this is needed. In classic tic-tac-toe one doesn't remove symbols from the board.